src/Homework6/parametrized_decorator.py

```diff
@@ -0,0 +1,44 @@
+"""Создайте декоратор, который вызывает задекорированную функцию
+ пока она не выполнится без исключений (но не более n раз -
+  параметр декоратора). Если превышено количество попыток,
+   должно быть возбуждено исключение типа TooManyErrors
+"""
+
+
+def count_errors(max_errors, runs):
+    def decorator(func):
+        def wrapper(*args, **kw):
+            calls = 0
+            errors = 0
+
+            class TooManyErrors(Exception):
```

Comment:
it would be better to define it outside of function - you can't catch such exception